demo-shell-ng2/app/components/sites/sites-demo.component.html

```diff
@@ -0,0 +1,12 @@
+<div class="demo-site-container-style" id="demo-container">
```

Comment:
why do we need a new page can not show this functionality in the current files.component?